### ISSUE TITLE
Issue with VBoxClient and display manager

### ISSUE THREAD

**Issue by carlwilson** [openpreserve/ddhn-forge#24]
This is a strange issue. When the VM is started the user is presented with a rough looking `X11` dialog with this text
```
VBoxClient: Failed to get display change request, rc=VERR_IO_BAD_LENGTH
```
which you'll need to dismiss. This has manifest itself as I've brought the PRs for the beta into a single Vagrant and Ansible build. None of the individual PRs break this individually, only when combined. The bad news is that this is hard to source, the good news is that once you've pressed the OK button you're good, though it seems to take a long time for this to happen when the machine is first started. It looks like this, I'll track and fix over the next week.

It looks like this:

![Kazam_screenshot_00016](https://user-images.githubusercontent.com/440634/92922989-9e50f480-f42e-11ea-817a-59d9f7f91f3c.png)

**Comment by Hello-gzk-World** [comment]
Did have fixed it?

**Comment by MomoMoses** [comment]
sorry I posted this same issue, but it was with an elaboration: it kept on popping up throughout my session

**Comment by Hello-gzk-World** [comment]
Thanks， me too

**Comment by vachmara** [comment]
I have the same issue. 3D acceleration in Setting/Display/Screen seems to be the cause of the problem. Turn off 3D acceleration work fine for me.

**Comment by carlwilson** [comment]
I believe that this will be fixed with #17, review if that is now the case.
